utils/cookies_login.py

```
@@ -0,0 +1,348 @@
+    """解析纯文本 Cookie"""
+    cookies = {}
+    if not cookie_string: return cookies
+    for item in cookie_string.split(';'):
+        if '=' in item:
+            key, value = item.strip().split('=', 1)
+            cookies[key.strip()] = value.strip()
+    return cookies
+
+ 
+
+async def get_bilibili_cookies(method: str = "manual") -> Optional[Dict[str, str]]:
```

Ruff (0.15.1)

[warning] Unused function argument: `method`

(ARG001)